src/app/screens/Home/index.js

```diff
@@ -0,0 +1,14 @@
+import clsx from 'clsx';
+import React from 'react';
+
+import Navbar from '~components/Navbar';
+
+import styles from './styles.module.scss';
+
+export default function Home() {
+  return (
+    <div className={clsx('column', styles.container)}>
+      <Navbar />
+    </div>
```

Comment:
i would create a router here for the private routes, and if there is no token here i would redirect to login